PySDM/physics/diffusion_kinetics/lowe_et_al_2019.py

```diff
@@ -2,25 +2,21 @@
 eq. 13-14 in Pruppacher & Klett 2005 for Delta v = 0
```

Comment:
perhaps let's add a comment based on this discussion: https://github.com/open-atmos/PySDM/pull/754#issuecomment-1032906497

Reply:
i.e., at least a mention of Okuyama & Zung 1967